### PR TITLE
v1

### PR DESCRIPTION
**Pull request by randyhoulahan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    "@vue/cli-plugin-babel": "^3.4.0",
+    "@vue/cli-plugin-e2e-cypress": "^3.4.0",
+    "@vue/cli-plugin-eslint": "^3.4.0",
+    "@vue/cli-service": "^3.4.0",
+    "babel-eslint": "^10.0.1",
+    "eslint": "^5.8.0",
+    "eslint-plugin-vue": "^5.0.0",
+    "vue-template-compiler": "^2.6.6"
+  },
+  "eslintConfig": {
```
Comment (blaisefonseca) [inline package.json:41]: why elslint config is in here?

File: scripts/mongo-index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    {
+      'organizer.name.en': "text",
+      'organizer.lastName.en': "text",
+      'organizer.email': "text",
+      'name.en': "text",
+      'description.en': "text",
+      'linkagesDescription.en': "text",
+      'progressMeasured.en':"text"
+    }
+  )
```
Comment (blaisefonseca) [inline scripts/mongo-index.js:11]: thi shld be in gaia initialize

File: src/components/AAForm/AAOrganizationForm.vue [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+  import SCBDSelect from './controls/SCBDSelect' 
+  import Links from './controls/Links'
+
+  export default {
+    name: 'AAOrganizationForm',
+    mixins: [AAFormMixin],
+    components: {  SCBDSelect, Links },
+    props: {
+      value: {
+        type: [Array, Object],
```
Comment (blaisefonseca) [inline src/components/AAForm/AAOrganizationForm.vue:138]: should this be not Array or Object?

Comment (randyhoulahan) [reply src/components/AAForm/AAOrganizationForm.vue:138]: that is what it means.  The list of types are any of the types provided.  The component can be used to select a single entry as an object or many as an array.

File: scripts/mongo-index.js (hunk 2 above)
Comment (randyhoulahan) [reply scripts/mongo-index.js:11]: yes, I could not think of a spot for it at the time so I just put it there to saving.

File: src/components/AAForm/controls/Links.vue [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+    startUploading,
+    getSrc
+
+  }
+}
+
+async function getPreSignedPost(file, comp) {
+  try {
+
+    let presignedPostUrl = 'https://88g0q0fw15.execute-api.us-east-1.amazonaws.com/dev/s3-pre-sign/upload'
```
Comment (blaisefonseca) [inline src/components/AAForm/controls/Links.vue:280]: temp url api

File: package.json (hunk 1 above)
Comment (randyhoulahan) [reply package.json:41]: the builder requires it.

File: src/components/AAForm/index.vue [hunk 5]
```diff
@@ -0,0 +1,387 @@
...
+ 
+        console.error(err);
+    }
+}
+async function getRecaptchaToken(options) {
+ 
+  if(!this.grecaptchaReady) return options
+  let token = await this.$recaptcha('action')
+  if(token )
+    options.headers['X-Captcha-Token'] = token
```
Comment (blaisefonseca) [inline src/components/AAForm/index.vue:341]: lowercase header

**Pull request closed without merging**